package.json

```diff
@@ -2,7 +2,7 @@
   "name": "silvermine-videojs-airplay",
   "version": "0.9.0",
   "description": "video.js plugin for casting to airplay",
-  "main": "src/index.js",
+  "main": "src/js/index.js",
   "scripts": {
     "test": "./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- -R spec 'tests/**/*.test.js'"
```

Comment:
Please add a `prepublish` script that builds the minified versions, as @onebytegone did at https://github.com/silvermine/videojs-quality-selector/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7